project/Game.py

```diff
@@ -420,23 +454,25 @@ def test_add_landmark(self):
 
 
 class landmarkDummy:
-    clue = "The Place we drink coffee and read books"
-    question = "What is the name of the statue out front?"
-    answer = "three disks"
-    location = "uwm library"
-    points = 150
-    timer = datetime.time(0,0,15)
-    timepenelty = 20
-    answerpenalty = 10
+    def __init__(self):
+        self.clue = "The Place we drink coffee and read books"
+        self.question = "What is the name of the statue out front?"
+        self.answer = "three disks"
+        self.location = "uwm library"
+        self.points = 150
+        self.timer = datetime.timedelta(hours=0,minutes=0,seconds=15)
+        self.timePenalty = 20
+        self.answerpenalty = 10
```

Comment:
There's a lot of new stuff here that isn't in the real Landmark class. Stuff should be implemented and tested in the base class before added to any dummies.